service/controller/v13/cloudconfig/interface.go

```diff
@@ -1,12 +1,14 @@
 package cloudconfig
 
 import (
+	"context"
+
 	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
 	"github.com/giantswarm/certs/legacy"
 	"github.com/giantswarm/randomkeys"
 )
 
 type Interface interface {
-	NewMasterTemplate(customObject v1alpha1.AWSConfig, certs legacy.CompactTLSAssets, clusterKeys randomkeys.Cluster, kmsKeyARN string) (string, error)
+	NewMasterTemplate(ctx context.Context, customObject v1alpha1.AWSConfig, certs legacy.CompactTLSAssets, clusterKeys randomkeys.Cluster) (string, error)
 	NewWorkerTemplate(customObject v1alpha1.AWSConfig, certs legacy.CompactTLSAssets) (string, error)
```

Comment:
My OCD thinks we should provide a context here as well. 🙈

Reply:
Done, not needed right now but good to have it in place.